src/rust/ide/src/view/node_editor.rs

```diff
@@ -145,7 +145,10 @@ impl GraphEditorIntegratedWithController {
 #[derive(Debug)]
 struct GraphEditorIntegratedWithControllerModel {
     logger             : Logger,
-    editor             : GraphEditor,
+    //TODO[ao] we display the new "Project view" because it contains documentation panel, but no
+    // text editor. This should be refactored as a part of task
+    // https://github.com/enso-org/ide/issues/597
+    view               : ide_view::project::View,
```

Comment:
So we have graph editor that contains project view that contains graph editor and documentation panel? 🤔